### ISSUE TITLE
doesn't works with PODMAN

### ISSUE THREAD

**Issue by bitchecker** [lionelnicolas/dalidock#7]
Hi,
`docker` it's been replaced by `podman` on new `Red Hat` systems so i'm trying to run this awesome tool on it for running some tests.

I'm trying to run on a libvirt server, but the error that I get is:
```
During handling of the above exception, another exception occurred:
                                               
Traceback (most recent call last):             
  File "/usr/sbin/dalidock", line 1110, in <module>
    main()                                                                             
  File "/usr/sbin/dalidock", line 1096, in main
    docker_evt_handler = DockerEventsHandler(dns_entries, lb_entries, config_lock)           
  File "/usr/sbin/dalidock", line 565, in __init__ 
    self.docker = DockerClient(DOCKER_SOCKET)                    
  File "/usr/sbin/dalidock", line 482, in __init__                
    self.showInformations()                                            
  File "/usr/sbin/dalidock", line 485, in showInformations
    docker_infos = self.conn.info()                                 
  File "/usr/lib/python3/dist-packages/docker/api/daemon.py", line 33, in info
    return self._result(self._get(self._url("/info")), True)          
  File "/usr/lib/python3/dist-packages/docker/utils/decorators.py", line 47, in inner
    return f(self, *args, **kwargs)                          
  File "/usr/lib/python3/dist-packages/docker/client.py", line 139, in _get
    return self.get(url, **self._set_request_timeout(kwargs))                            
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)         
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)                          
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))
```

For run this, there are some options that doesn't works, so, my CLI was:
```
podman run \
   --name dalidock \
   --net bridge \
   --log-opt max-size=50m \
   --log-opt max-file=3 \
   --cap-add NET_ADMIN \
   --publish $_IP_ADDRESS_:53:53/udp \
   --publish 80:80 \
   --env DNS_DOMAIN=my.local.env \
   --env LB_DOMAIN=my.local.env \
   --volume /run/NetworkManager:/run/NetworkManager:ro \
   --volume /var/run/libvirt:/var/run/libvirt:ro \
   lionelnicolas/dalidock
```
I don't used `--detached` for see what happens, so i see that it detects `libvirt` engine
```
dalidock[3798]: [INFO]  ----- LIBVIRT INFOS -----
[...]
```

So i think that for that errors, if i try to reach server via `telnet` to see if DNS service is running, i obtain `Connection timed out` also if via `netstat` i see correct bind `$_IP:53`

**Comment by lionelnicolas** [comment]
@bitchecker Thanks for reporting the issue.

That should be fixed in #12 . I'll need to spend more time to add support for `podman` API to discover these containers, but at least you should now be able to use `dalidock` with `libvirt`.

I also took the opportunity to fix various bugs and update dependencies and documentation ( #14 ). I'm using this project every day on my workstation, but I don't really spend time maintaining it.

I've pushed both `lionelnicolas/dalidock:latest` and  `lionelnicolas/dalidock:0.6.0` tags on dockerhub if you want to try.

**Comment by bitchecker** [comment]
Hi @lionelnicolas,
thanks for fast reply!
